### PR TITLE
i18N: sync from Transifex

### PR DESCRIPTION
**Pull request by neteler**
Source: https://www.transifex.com/grass-gis/grass7/dashboard/

via https://github.com/OSGeo/grass-addons/blob/master/tools/transifex_merge.sh

### CODE REVIEW

**Comment by neteler** [comment]
(Note: no backport wanted, that is handled via #558 )

File: locale/po/grasswxpy_cs.po [hunk 1]
```diff
@@ -20821,3 +20821,6 @@ msgstr "&Stáhnout"
 
 msgid "Fetch list of available modules from GRASS Addons SVN repository"
 msgstr "Stáhnout seznam dostupných modulů z GRASS Addons SVN repozitáře"
+
+#~ msgid "Animation already axists"
```
Comment (nilason) [inline locale/po/grasswxpy_cs.po:20825]: Should be “exists”, no?

Comment (neteler) [reply locale/po/grasswxpy_cs.po:20825]: In theory yes, but in practice it is dead old gettext message code as being `#~ msgid`

Comment (nilason) [reply locale/po/grasswxpy_cs.po:20825]: Ok, I assumed it maybe was an out-of-sync artefact.

Comment (neteler) [reply locale/po/grasswxpy_cs.po:20825]: Thanks for checking!

**Pull request merged**